### DISCUSSION TITLE
Support for XPU devices

### DISCUSSION THREAD

**Discussion by flezaalv** [team-ocean/veros#630] [General]
Is there a plan to support XPU devices?

**Comment by dionhaefner** [comment]
"Supporting" XPU should be a one-line change, simply add it here as a possible device: https://github.com/team-ocean/veros/blob/f80b6361cf7470579a769d1dfbe448db51c3d29a/veros/runtime.py#L80

That said no one in the dev team has an XPU device so there won't be official support unless that changes.
